BlueSlug/wecount.inclusivedesign.ca
# feat: remove "toolPurposes" and "toolAccessibilityIssues" metadata from resources collection

## Description

Remove "toolPurposes" and "toolAccessibilityIssues" metadata from resources collection.

## Steps to test

1. Go to resources and resources detail pages. The data from these two metadata should no longer be there.
2. Go to admin page -> update an existing resource -> check the saved .md file -> these two metadata should not show up.
3. Go to admin page -> create a new resource -> check the saved .md file -> these two metadata should not show up.

**Expected behavior:** 
See above.
